varats-core/varats/project/project_util.py

```diff
@@ -34,19 +35,50 @@ def get_primary_project_source(project_name: str) -> bb.source.BaseSource:
     return bb.source.primary(*project_cls.SOURCE)
 
 
-def get_local_project_git_path(project_name: str) -> Path:
+def get_local_project_git_path(
+    project_name: str, git_name: tp.Optional[str] = None
+) -> Path:
     """Get the path to the local download location of git repository for a given
     benchbuild project."""
-    primary_source = get_primary_project_source(project_name)
-    if hasattr(primary_source, "fetch"):
-        primary_source.fetch()
 
-    return Path(target_prefix() + "/" + primary_source.local)
+    if git_name:
+        source = get_extended_commit_lookup_source(project_name, git_name)
+    else:
+        source = get_primary_project_source(project_name)
 
+    if hasattr(source, "fetch"):
+        source.fetch()
 
-def get_local_project_git(project_name: str) -> pygit2.Repository:
+    return Path(target_prefix() + "/" + source.local)
```

Comment:
```suggestion
    return Path(target_prefix()) / source.local
```
You don't need to put everything together and the create a path. It's easier to use the path extender syntax with the `/` operator.